cumulus_library/databases/base.py

```diff
@@ -208,6 +205,16 @@ def upload_file(
         have an API for file upload (i.e. cloud databases)"""
         return None
 
+    @abc.abstractmethod
+    def export_table_as_parquet(
+        self, table_name: str, table_type: str, location: pathlib.Path, *args, **kwargs
```

Comment:
nit: `table_type` seems like an odd addition here. In both implementations, it's just used to create the output file name, which felt like a bit of duplicated business logic.

What if you used a "input/output, did it write anything" pattern like (totally ignorable suggestion, just brainstorming):
```
def export_table_as_parquet(self, table_name: str, output_path: pathlib.Path) -> bool:
```

Reply:
ok, as discussed offline, this now takes a `filename` arg and returns a bool